frontend/src/lib/components/PropertyFilters/components/TaxonomicPropertyFilter.tsx

```diff
@@ -85,7 +86,11 @@ export function TaxonomicPropertyFilter({
     const showInitialSearchInline =
         !disablePopover &&
         ((!filter?.type && (!filter || !(filter as any)?.key)) || filter?.type === PropertyFilterType.HogQL)
-    const showOperatorValueSelect = filter?.type && filter?.key && filter?.type !== PropertyFilterType.HogQL
+    const showOperatorValueSelect =
+        filter?.type &&
+        filter?.key &&
+        !(filter?.type === PropertyFilterType.HogQL) &&
+        !(filter?.type === PropertyFilterType.Cohort && exactMatchFeatureFlagCohortOperators)
```

Comment:
I use this prop to get rid of the operator value selector so we can have the same experience that we had before Sandy's change went in.